Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -751,6 +879,26 @@ void vtkMRMLSliceNode::WriteXML(ostream& of, int nIndent)
       }
     }
   of << indent << " sliceToRAS=\"" << ss.str().c_str() << "\"";
+
+  std::map<vtkStdString, vtkMatrix4x4*>::iterator it;
+  for (it = this->OrientationMatrices.begin(); it != this->OrientationMatrices.end(); ++it)
+    {
+    std::stringstream ss;
+    int j;
+    for (i=0; i<4; i++)
```

Comment:
You can add a method to the name-matrix struct (or class) that sets the vtkMatrix3x3 content based on a vtkMatrix4x4 input.